pkg/disk/nodeserver.go

```diff
@@ -990,14 +991,14 @@ func (ns *nodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 	deviceCapacity := getBlockDeviceCapacity(devicePath)
+	if deviceCapacity < requestGB {
+		// After calling openapi to expansion cloud disk, the size of the underlying block device may not change. need to retry
+		log.Errorf("NodeExpandVolume:: Actual block size: %s is smaller than expected block size: %s, need to retry waiting", deviceCapacity, requestGB)
+		return nil, status.Errorf(codes.Aborted, "deviceCapacity: %v, requestGB: %v not in range", deviceCapacity, requestGB)
+	}
 	log.Infof(
 		"NodeExpandVolume:: filesystem resize context device capacity: %v, before resize fs capacity: %v resize fs capacity: %v, requestGB: %v. file system lose percent: %v",
 		deviceCapacity, beforeResizeDiskCapacity, diskCapacity, requestGB, GlobalConfigVar.FilesystemLosePercent)
```

Comment:
Remove all codes about `FilesystemLosePercent`?

Reply:
yes, I don't think It serves its purpose.